test/test_default_api.py

```diff
@@ -903,6 +903,338 @@ def test_update_an_incoming_number(self):
+        
+        api_response = self.api.get_ten_dlc_sms_partner_campaigns(brand_id=brand_id)
+        api_response.next_page_uri = "/Accounts/{accountId}/Messages/10DLC/PartnerCampaigns?cursor=1"
+        next_page_api_response = self.api.get_next_page(api_response)
+
+        assert isinstance(next_page_api_response, SMSTenDLCPartnerCampaignsListResult)
+
+    def  test_get_toll_free_sms_campaigns_get_next_page(self):
+        """Test case for get_next_page using get_toll_free_sms_campaigns response
+
+        Get Next Page for Get list of TollFree Campaigns  # noqa: E501
+        """
+
+        
+        api_response = self.api.get_toll_free_sms_campaigns()
+        api_response.next_page_uri = "/Accounts/{accountId}/Messages/TollFree/Campaigns?cursor=1"
+        next_page_api_response = self.api.get_next_page(api_response)
+
+        assert isinstance(next_page_api_response, SMSTollFreeCampaignsListResult)
+
+    def  test_list_active_queues_get_next_page(self):
+        """Test case for get_next_page using list_active_queues response
+
+        Get Next Page for List Active Queues  # noqa: E501
+        """
+
+        alias = alias_list_active_queues_test_value
+        alias = alias_list_active_queues_test_value
+        
+        api_response = self.api.list_active_queues(alias=alias)
+        api_response.next_page_uri = "/Accounts/{accountId}/Queues?cursor=1"
+        next_page_api_response = self.api.get_next_page(api_response)
+
+        assert isinstance(next_page_api_response, QueueList)
+
+    def  test_list_all_account_logs_get_next_page(self):
+        """Test case for get_next_page using list_all_account_logs response
+
+        Get Next Page for List All Account Logs  # noqa: E501
+        """
+
+        
+        api_response = self.api.list_all_account_logs()
+        api_response.next_page_uri = "/Accounts/{accountId}/Logs?cursor=1"
+        next_page_api_response = self.api.get_next_page(api_response)
+
+        assert isinstance(next_page_api_response, LogList)
+
+    def  test_list_applications_get_next_page(self):
+        """Test case for get_next_page using list_applications response
+
+        Get Next Page for List applications  # noqa: E501
+        """
+
+        alias = alias_list_applications_test_value
+        alias = alias_list_applications_test_value
+        
+        api_response = self.api.list_applications(alias=alias)
+        api_response.next_page_uri = "/Accounts/{accountId}/Applications?cursor=1"
+        next_page_api_response = self.api.get_next_page(api_response)
+
+        assert isinstance(next_page_api_response, ApplicationList)
+
+    def  test_list_available_numbers_get_next_page(self):
+        """Test case for get_next_page using list_available_numbers response
+
+        Get Next Page for List available numbers  # noqa: E501
+        """
+
+        phone_number = phone_number_list_available_numbers_test_value
+        phone_number = phone_number_list_available_numbers_test_value
```

Comment:
Is there a reason why this is being generated twice?

Reply:
good catch, had a duplicated mustache line